src/lib/util/setOptions.ts

```diff
@@ -22,7 +22,7 @@ export const setOptions = (options: Options): Options => {
 
   if (options.name) {
     const tags = clusterOptions.node_pools[0].tags;
-    clusterOptions.node_pools[0].name = options.name;
+    clusterOptions.node_pools[0].name = options.name.slice(-55);
```

Comment:
Let's set -55 as a variable or add a comment giving it some context.  Just something where we can note it's not just some arbitrary number.